microsoft-edge/extensions-chromium/developer-guide/api-support.md

```diff
@@ -11,57 +11,90 @@ ms.date: 08/28/2023
+| [notifications](https://developer.chrome.com/docs/extensions/reference/api/notifications) | Create rich notifications using templates and display them in the system tray. |
+| [omnibox](https://developer.chrome.com/docs/extensions/reference/api/omnibox) | Register keywords in the Microsoft Edge address bar, also known as the omnibox. |
+| [pageAction](https://developer.chrome.com/docs/extensions/mv2/reference/pageAction) | todo: MV2, not MV3.  Add icons to the Microsoft Edge toolbar, to the right of the address bar. Page actions are actions that can be taken on the current page, and aren't applicable to all pages. Page actions appear grayed out when inactive. |
+| [pageCapture](https://developer.chrome.com/docs/extensions/reference/api/pageCapture) | Save tabs as MHTML files.|
+| [permissions](https://developer.chrome.com/docs/extensions/reference/api/permissions) | Retrieve declared, optional permissions at runtime, instead of at install time. You can use this API to display needed and approved permissions to your users. |
+| [platformKeys](https://developer.chrome.com/docs/extensions/reference/api/platformKeys) | todo: supported? |
+| [power](https://developer.chrome.com/docs/extensions/reference/api/power) | Override the system's power management features. |
+| [printerProvider](https://developer.chrome.com/docs/extensions/reference/api/printerProvider) | Use events to query printers, their capabilities, and to submit print jobs. |
+| [printing](https://developer.chrome.com/docs/extensions/reference/api/printing) | todo: supported? |
+| [printingMetrics](https://developer.chrome.com/docs/extensions/reference/api/printingMetrics) | todo: supported? |
+| [privacy](https://developer.chrome.com/docs/extensions/reference/api/privacy) | Control features in Microsoft Edge that affects a user's privacy. This API depends on the `EdgeSetting` prototype of `types` to get and set the configuration of Microsoft Edge. |
+| [processes](https://developer.chrome.com/docs/extensions/reference/api/processes) | todo: supported? |
+| [proxy](https://developer.chrome.com/docs/extensions/reference/api/proxy) | Manage proxy settings for Microsoft Edge. This API depends on the `EdgeSetting` prototype of the `types` API to get and set the proxy configuration of Microsoft Edge. |
+| [runtime](https://developer.chrome.com/docs/extensions/reference/api/runtime) | Retrieve the background page, return details about the manifest, and listen for and respond to events in the app or extension lifecycle. You can also convert the relative path of URLs to fully qualified URLs. |
+| [scripting](https://developer.chrome.com/docs/extensions/reference/api/scripting) | todo: supported? |
+| [search](https://developer.chrome.com/docs/extensions/reference/api/search) | todo: supported? |
```

Comment:
@Bhuvana-Priya -- "search" is an API added by this PR.  Is it supported for extensions?

"search" api is listed for MV2 & MV3 at Chrome:
MV2 Extensions API Ref page:
https://developer.chrome.com/docs/extensions/mv2/reference/
MV3 Extensions API Ref page:
https://developer.chrome.com/docs/extensions/reference/api/

"search" is not listed in the "not supported" section below.

Reply:
in such case, Yes, where not marked as "ChromeOS only"